src/number.ts

```diff
@@ -1,31 +1,30 @@
 import { formatFixed } from "@ethersproject/bignumber"
-import { BigNumber } from "ethers"
 
 export interface HardhatNumberHelpers {
-  to1e18(n: any): BigNumber
-  to1ePrecision(n: any, precision: number): BigNumber
+  to1e18(n: any): bigint
+  to1ePrecision(n: any, precision: number): bigint
   from1e18(n: any): string
   from1ePrecision(n: any, precision: number): string
 }
 
-export function to1e18(n: any): BigNumber {
+export function to1e18(n: any): bigint {
   return to1ePrecision(n, 18)
 }
 
-export function to1ePrecision(n: any, precision: number): BigNumber {
-  const decimalMultiplier = BigNumber.from(10).pow(precision)
-  return BigNumber.from(n).mul(decimalMultiplier)
+export function to1ePrecision(n: bigint, precision: number): bigint {
```

Comment:
We could improve the `to1e18` and `to1ePrecision` functions and allow `string | number | bigint` as `n` input.
```ts
export function to1ePrecision(
  n: string | number | bigint,
  precision: number,
): bigint {
  return BigInt(n) * 10n ** BigInt(precision)
}
```

Reply:
647ceed - used `BigNumberish` type as this is doing the same thing